setup.py

```diff
@@ -32,5 +32,8 @@
         "PyYAML >= 3.0",
         "pyOpenSSL >= 19.1.0",
     ],
-    extras_require={"tests": tests_requirements},
+    extras_require={
+        "fs": ["fsspec == 2021.07.0; tqdm >= 4.0.0;"],
```

Comment:
Are there any other packages that have such option? To me only webdav4 comes to mind, so maybe 
```suggestion
        "fsspec": ["fsspec == 2021.07.0; tqdm >= 4.0.0;"],
```
?